app/models/building.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Model to represent the highest-level of facility information
```

Comment:
These comments are all good (although mildly inconsistent in terms of punctuation :stuck_out_tongue:). Can we make sure that they fit into a standard Ruby documentation format, e.g. [RDoc](https://github.com/rdoc/rdoc) or [YARD](http://yardoc.org/)? If we use YARD we can actually host our documentation [here](http://www.rubydoc.info/) if we wanted (not necessary).